browser/brave_profile_prefs_browsertest.cc

```diff
@@ -82,17 +82,28 @@ typedef FirstRunMasterPrefsBrowserTestT<kFirstRunEmptyPrefs>
     BraveProfilePrefsFirstRunBrowserTest;
 IN_PROC_BROWSER_TEST_F(BraveProfilePrefsFirstRunBrowserTest,
                        AdvancedShieldsNewUserValue) {
-  EXPECT_FALSE(
-        browser()->profile()->GetPrefs()->GetBoolean(
+  // verify value of pref (default to simple view)
+  EXPECT_FALSE(browser()->profile()->GetPrefs()->GetBoolean(
           kShieldsAdvancedViewEnabled));
+  // verify that pref was set (and is not default)
+  const PrefService::Preference* pref =
+      browser()->profile()->GetPrefs()->FindPreference(
+          kShieldsAdvancedViewEnabled);
+  EXPECT_TRUE(pref->HasUserSetting());
 }
 
 // Existing Brave users should default shields to Advanced view
 IN_PROC_BROWSER_TEST_F(BraveProfilePrefsBrowserTest,
                        AdvancedShieldsExistingUserValue) {
+  // verify value of pref (default to advanced view)
   EXPECT_TRUE(
         browser()->profile()->GetPrefs()->GetBoolean(
           kShieldsAdvancedViewEnabled));
+  // verify that pref was set (and is not default)
```

Comment:
I think this test is redundant with above one? what is the purpose of this test?
Each browser tests uses fresh profile.
I assume you want spanned browser test? (https://www.chromium.org/developers/testing/browser-tests)

Reply:
Goal for this test is to have a non-first-run profile and verify that value does get set and that it's what we expect (advanced) 😄 

Versus the one above, which is having an empty first run profile and then verifying it gets simple view

Reply:
Oh, I didn't know profile that used for this test is non-first-run profile :) 
I thought all browser tests uses fresh test profile.